vello_encoding/src/ramp_cache.rs

```diff
@@ -35,13 +36,14 @@ impl RampCache {
     }
 
     pub(crate) fn add(&mut self, stops: &[ColorStop]) -> u32 {
-        if let Some(entry) = self.map.get_mut(stops) {
+        if let Some(entry) = self.map.get_mut(&CacheKey(ColorStops(stops.into()))) {
```

Comment:
This is not ideal, as it allocates "unnecessarily". It really complicates the traits in the Color crate to make this work nicely, I think.